grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractServerCall.java

```diff
@@ -540,7 +540,14 @@ static HttpHeaders statusToTrailers(
         GrpcTrailersUtil.addStatusMessageToTrailers(
                 trailersBuilder, status.getCode().value(), status.getDescription());
 
-        MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        try {
```

Comment:
What do you think of moving this statement before L540?
The line above may add a message which may turn out to be invalid if an exception is reached here

Reply:
Is it possible that with an invalid metadata, the status is overridden but the message is not?

https://github.com/line/armeria/blob/be306ab4c9de5386e74999baec5ebaae82618b91/grpc-protocol/src/main/java/com/linecorp/armeria/internal/common/grpc/protocol/GrpcTrailersUtil.java#L50

Reply:
Oh, I see. For the consistency. This makes sence.